src/fusionAuth.ts

```diff
@@ -8,6 +8,8 @@ export const getFusionAuthClient = (): FusionAuthClient => new FusionAuthClient(
 export interface PartialClientResponse {
   exception: {
     message: string;
+    error?: string;
```

Comment:
Just a sanity check: is it possible for FusionAuth to have an exception without an error / error_description?

Would it make sense to have a separate type / interface of `PartialClientErrorResponse` that more explicitly defines what we're expecting?

Reply:
Yes, sometimes they return just the message. I checked their response interface and `error` is optional, I wonder why they would do that.

Reply:
Looking at your second comment, this is most likely because I was catching the `clientResponse` and not the exception, but I wonder if it makes any difference because at the end `clientResponse` is just a function-level scoped var name.

Reply:
Right.  Regarding that second point: you're catching any possible exception -- client response is just one type of exception, which is why `clientResponse` is an inaccurate name for that variable name since it could be anything (hence being unknown!)

So I'd rename it to `err` and then have type-check based handling for anticipated error types (e.g. the client response), and wrap up with a generic "unknown error" response.